### PR TITLE
Stop reverting on precompiles

### PR DESCRIPTION
**Pull request by ClementWalter**
Time spent on this PR: 0.2

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Some Cairo VM errors in precompiles

## What is the new behavior?

Returns a reverted context as in other errors.

### CODE REVIEW

**Review by Eikix — APPROVED** [review]
lgtm

File: src/kakarot/errors.cairo [hunk 1]
```diff
@@ -499,4 +499,67 @@ namespace Errors {
         dw 'a';
         dw 's';
     }
+
+    func precompileInputError() -> (error_len: felt, error: felt*) {
```
Comment (greged93) [inline src/kakarot/errors.cairo:503]: should the error take as input the precompile number and append it to `error`?

File: src/kakarot/errors.cairo [hunk 2]
```diff
@@ -499,4 +499,67 @@ namespace Errors {
...
+        dw 'p';
+        dw 'u';
+        dw 't';
+        dw '_';
+        dw 'l';
+        dw 'e';
+        dw 'n';
+    }
+
+    func precompileFlagError() -> (error_len: felt, error: felt*) {
```
Comment (greged93) [inline src/kakarot/errors.cairo:537]: same as above?

**Review by greged93 — APPROVED** [review]
small suggestion, can be added or not

File: src/kakarot/errors.cairo (hunk 1 above)
Comment (ClementWalter) [reply src/kakarot/errors.cairo:503]: And well, I've guessed that this logic was not worth it

**Pull request merged**